pages/advanced-algorithms/deep-path-traversal.mdx

```diff
@@ -346,6 +346,110 @@ MATCH path=(n {id: 0})-[*BFS (r, n, p | r.eu_border = false AND n.drinks_USD < 1
 RETURN path;
 ```
 
+## Constraining the expansion based on the number of hops
+
+### Overview
+Breadth-First Search (BFS), Depth-First Search (DFS), and expand query traversals can now be controlled by the number of hops (edges crossed). 
+Each unit of traversal, including filtering on edge types, edge properties, or visiting already processed nodes, is counted as a hop.
+
+<Callout type="warning">
+
+The hops limit does not affect the Weighted Shortest Path, All Shortest Paths, and expansions where relationships are created (CreateExpand operator).
+
+</Callout>
+
+### Syntax Usage
+The `USING HOPS LIMIT x` syntax is implemented as a query directive, similar to [Index Hints](/fundamentals/indexes#index-hinting). Users can apply the hops limit as follows:
+
+```cypher
+USING HOPS LIMIT x ...;
+```
+
+The hops limit is global for the entire query and can only be defined once. If you need to use both index hints and a hops limit, the directives can be chained:
+
+```cypher
+USING HOPS LIMIT x, :Label(property) ...;
+```
+
+### Implementation Variations
+
+<Callout type="warning">
+
+Keep in mind number of hops throughout the query could be different in different algorithms, due to differences in the implementation of different operators.
+Also, Memgraph uses two different operators for bfs due to optimization reasons, and even with them, the number of hops can differ for the same paths.
+
+</Callout>
```

Comment:
Remove  `type="warning"` from `Callout`. It is not needed in this case.

Remove word `Also`.